frontend21/scss/components/extra/about/components-showcase.scss

```diff
@@ -1,23 +1,215 @@
-.#{project('components-showcase')} {
-  background: red;
+.#{project("components-showcase")} {
+  position: relative;
+  height: 100%;
 
   &-buttons {
-    background: lime;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 0.625rem;
+    margin-bottom: 1.875rem;
+    button {
+      margin-bottom: 0;
+    }
   }
 
   &-demos {
-    background: cyan;
+    .show {
+      position: relative;
+      display: block;
+      height: 455px;
+    }
+
+    .hide {
+      display: none;
+    }
   }
 
   &-demo {
-    background: yellow;
+    position: relative;
 
     &-code {
-      background: slategray;
+      @include fill;
+      top: 20px;
+      width: 64.9%;
+      height: 100%;
+      max-height: 420px;
+
+      &-snippet {
+        height: 92%;
+      }
+
+      &:before {
+        content: attr(title);
+        position: absolute;
+        bottom: calc(100% - 20px);
+        width: calc(100% - 40px);
+        padding: 12px 20px;
+        border-bottom: 2px solid color("river-bed");
+        background: color("charade");
+        color: white;
+        font-family: $fontFamilyRegular;
```

Comment:
Could it be that `$fontFamilyRegular` has the wrong value? I think it needs to be `'Noto', system` instead of `'Noto Sans', system` - only then the title looks correct.

Reply:
Hm it is 'Noto' system here. But I started had started with the `npm run start:frontend`. Without it is Noto Sans. 
But I rechecked with the design and it is actual Poppins. So I'll change it to `$fontFamilyHeadline`